connexion/decorators/validation.py

```diff
@@ -71,15 +70,13 @@ def make_type(value, type_literal):
         if param_schema.get('properties'):
             def cast_leaves(d, schema):
                 if type(d) is not dict:
-                    try:
-                        return make_type(d, schema['type'])
-                    except (ValueError, TypeError):
-                        return d
+                    return coerce_type(schema, d, parameter_type)
                 for k, v in d.items():
                     if k in schema['properties']:
                         d[k] = cast_leaves(v, schema['properties'][k])
+                    elif schema.get('additionalProperites'):
```

Comment:
how does this handle an explicit `additionalProperties: true`?
It looks like it calls `cast_leaves` -> `coerce_type` so `param` ends up as a True bool, so `param.get("schema", param)` (line 54) should fail.
If `additionalProperties` is not defined or None (ie the default value of True applies), then this skips handling it.
```suggestion
                    elif schema.get('additionalProperites') and not isinstance(schema.get('additionalProperties'), bool):
```
But that doesn't handle cases where `additionalProperties` are just allowed, and no schema for them is defined.

Reply:
also spelling - 'additionalPrope**rites**' --> 'additionalPrope**rties**'